### ISSUE TITLE
When the contents of Dialog exceeds the computer screen,the browser scroll bar does not work

### ISSUE THREAD

**Issue by smart-dream** [likeastore/ngDialog#521]
When the contents of Dialog exceeds the computer screen, in addition to the contents of Dialog by rolling the mouse can see more than part of the content, other operations can not see beyond the computer screen part of the content (eg: 1. mouse point to the right of the scroll bar to drag, can not operate; In the Dialog content area is not a rolling mouse page also no change, can not see beyond the content.) In the case (http://likeastore.github.io/ngDialog/#) also tried to be longer content area, there is this problem. Please help me to see , very grateful!

**Comment by DerZyklop** [comment]
Not sure if i understood you correctly but i think i have the same issue. 

Scrolling only works when the mouse is hovering the popup itself. 
Scrolling does not work when the mouse hovers `.ngdialog-overlay`.

**Comment by DerZyklop** [comment]
I fixed it for me by adding these lines to my CSS:

    .ngdialog-overlay {
    	display: none;
    }
    .ngdialog {
    	background: rgba(0, 0, 0, 0.4);
    	animation: ngdialog-fadein 0.5s;
    }

**Comment by dpicode** [comment]
Hi, with the previous comment of @DerZyklop , you can fix the problem, but if you click out of the dialog, didn't close the dialog and this maybe generate you another user experience mistake.

Using the idea from @DerZyklop, we've fixed the issue. 
You can see the correction here: 
https://github.com/likeastore/ngDialog/pull/553

**Comment by burdell** [comment]
@dpicode Heads up, I can see the fixes in the `1.2.0` release, but don't see them in the  `1.3.0`!

**Comment by dmitry-zaets** [comment]
@burdell @dpicode Yep, It's not in master anymore: https://github.com/likeastore/ngDialog/compare/master...dpicode:patch-1?expand=1

Related issues and PRs
(https://github.com/likeastore/ngDialog/issues/552, https://github.com/likeastore/ngDialog/issues/532)

**Comment by dpicode** [comment]
Done, we reapplied the changes. 
We don't know what happened with the merge.

**Comment by dmitry-zaets** [comment]
This fix is causing an inconsistent UI animation behavior because the layers get animated with different timings. If you will check the examples you will be able to see that first the modal window disappears and then the background, but expected behavior is that both disappears at the same time.

Reproducible here: https://github.com/likeastore/ngDialog/blob/master/example/index.html
Expected behavior: http://likeastore.github.io/ngDialog/
